package.json

```diff
@@ -4308,6 +4308,19 @@
 					}
 				}
 			},
+			{
```

Comment:
Change to `9000` to show up before Advanced?

Reply:
Sounds good.

Reply:
Done in commit: https://github.com/gitkraken/vscode-gitlens/commit/651c5adf49adcdbeab5080fb52a14937a5d5c3a2